src/codeview.js

```diff
@@ -220,6 +231,18 @@ var Codeview = GObject.registerClass({
         this._userFunction = '';
     }
 
+    get errors() {
+        return this._numErrors;
+    }
+
+    set errors(value) {
+        this._numErrors = value;
```

Comment:
Return/skip if `this._numErrors === value`, otherwise a property change will be emitted in vain.